articles/MeetUsers

```diff
@@ -0,0 +1,214 @@
+# meet with users - often
```

Comment:
Titles should be capitalized: Meet With Users - Often
(as usual in English)
see also http://www.businesswritingblog.com/business_writing/2010/02/words-to-capitalize-in-titles-and-headings.html - maybe we have to define which exact capitalization rules we want to enforce in the book